api/src/main/java/com/datastrato/gravitino/rel/TableChange.java

```diff
@@ -581,9 +583,7 @@ final class After implements ColumnPosition {
     private final String column;
 
     private After(String column) {
-      if (null == column) {
-        throw new IllegalArgumentException("column can not be null.");
-      }
+      Preconditions.checkArgument(!Strings.isNullOrEmpty(column), "Column cannot be null");
```

Comment:
Is this an unrelated change about `UseCorrectAssertInTests`?

Reply:
@qqqttt123, #2362 mentions this change as `We'd better using Guava's Preconditions, rather than if..else..., which is not elegant.`. Meanwhile, I have mentioned this change in description.

Reply:
I prefer not including this change in this pr.

Reply:
I'm okay with it.